src/app/modules/submission/encrypt-submission/encrypt-submission.utils.ts

```diff
@@ -154,7 +154,7 @@ const errorMapper: MapRouteError = (
       }
     case VerifyTurnstileError:
       return {
-        statusCode: StatusCodes.BAD_REQUEST,
+        statusCode: StatusCodes.INTERNAL_SERVER_ERROR,
```

Comment:
Similar argument as the client vs server error [above](https://github.com/opengovsg/FormSG/pull/6494/files/5e732d7da418aa06c939e0e60a8b4e3af030fe1c..af4156589ef176a787c2284f9b2cf6dfecf94769#r1246303975).

Reply:
Same as above